test/rmq/jupyter-test.py

```diff
@@ -0,0 +1,23 @@
+# %%
+
+import kiwipy
+
+comm = kiwipy.connect('amqp://127.0.0.1')
+
+
+def rpc_call(_comm, msg):
+    print("Got your message: {}".format(msg))
+    return "Thanks!"
+
+
+comm.add_rpc_subscriber(rpc_call, 'tester')
+
+response = comm.rpc_send('tester', 'hello!')
+
+# %%
+
+print(response.result())
```

Comment:
Do we maybe want to turn this into an assert on the actual expected response?

Reply:
I created this script and the corresponding ipy notebook because they both do the same thing but the jupyter notebook used to fail because of the event loop issues.  I'm not sure if/how these could be executed manually but I don't see the effort/reward ratio being big enough to warrant setting that up given that this specific bug is unlikely to reappear in this form again.  It is good to have them around though so they can be manually invoked just in case.